server/liveblog/blogs/blogs.py

```diff
@@ -83,21 +83,13 @@ class BlogsVersionsResource(ArchiveVersionsResource):
     """
 
     datasource = {
-        'source': 'archive' + '_versions',
-        'filter': {'type': 'composite'}
+        'source': 'archive' + '_versions'
     }
 
 
-class BlogsVersionsService(BaseService):
-    def get(self, req, lookup):
-        if req is None:
-            req = ParsedRequest()
-        return self.backend.get('archive_versions', req=req, lookup=lookup)
-
-
 class BlogsResource(ArchiveResource):
     datasource = {
-        'source': 'archive',
```

Comment:
Either you let 'search_backend': 'elastic' or 'source': 'blogs' it will use as source the "blogs" collections. In order to keep things homogeneous we can keep   'source': 'blogs'.